.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -23,32 +23,18 @@ assignees: ''
 
 ## Steps to Reproduce
 
-<!--
-     Please tell us exactly how to reproduce the problem you are running into.
-
-     Please attach a small application (ideally just one main.dart file) that
-     reproduces the problem. You could use https://gist.github.com/ for this.
-
-     If the problem is with your application's rendering, then please attach
-     a screenshot and explain what the problem is.
--->
+<!-- You must include full steps to reproduce so that we can reproduce the problem. -->
 
-1. ...
-2. ...
-3. ...
+1. Run `flutter create bug`.
+2. Update the files as follows: ... <!-- include every file that is different from the template app! -->
+3. ... <!-- describe how to reproduce the problem -->
 
-<!--
-     Please tell us which target platform(s) the problem occurs (Android / iOS / Web / macOS / Linux / Windows)
-     Which target OS version, for Web, browser, is the test system running?
-     Does the problem occur on emulator/simulator as well as on physical devices?
--->
+**Expected results:** <!-- what did you see? -->
 
-**Target Platform:**
-**Target OS version/browser:**
-**Devices:**
+**Actual results:** <!-- what did you want to see? -->
```

Comment:
Maybe this comment is for **Expected results:**?

Reply:
Good catch -- yeah, looks like this is flipped around.